src/app/components/search-bar/search-bar.component.scss

```
@@ -81,6 +84,19 @@
     transform: translateY(-2rem);
   }
 
+  .filter-draft-input--compact.filter-draft-input--dirty {
+    @apply border-solid border-yellow-500;
+    background-color: rgb(234 179 8 / 0.15);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)